resources/views/home.blade.php

```diff
@@ -1,23 +1,48 @@
 @extends('layouts.app')
 
 @section('content')
-<div class="container">
-    <div class="row justify-content-center">
-        <div class="col-md-8">
-            <div class="card">
-                <div class="card-header">Dashboard</div>
+    <div class="container">
+        <div class="row justify-content-center">
+            <div class="col-md-8">
+                <div class="card">
+                    <div class="card-header">Dashboard</div>
 
-                <div class="card-body">
-                    @if (session('status'))
-                        <div class="alert alert-success" role="alert">
-                            {{ session('status') }}
-                        </div>
-                    @endif
+                    <div class="card-body">
+                        @if (session('status'))
+                            <div class="alert alert-success" role="alert">
+                                {{ session('status') }}
+                            </div>
+                        @endif
+                        
+                        Your boulders:
+                        <br/> <br/>
 
-                    You are logged in!
+                        <u>Finished boulders:</u> <br/>
+                        @foreach($boulders as $boulder)
+                            @if($boulder->completed_at != null)
```

Comment:
It's always better to use !== and === if you expect one specific thing.
Because PHP automatically casts variables for checking otherwise.

If you don't want to check something specific but just want to know of 'completed_at' is a falsy value, you could do: @if(!$boulder->completed_at)

Reply:
It might also be nicer to split up the boulders into two array before you send them to the view. But that's not too important, but it's a possibility.

Reply:
i was thinking of the second one as well with also just providing the information needed but in a later state